test/wallet/index.ts

```diff
@@ -199,9 +217,278 @@ describe('Wallet', function () {
       const wallet = new Wallet(publicKey, privateKey)
       const signedTx: string = wallet.signTransaction(txJSON)
 
-      // TODO: Check the output of the signature against a known result
       assert.isString(signedTx)
     })
+
+    it('sign', async function () {
```

Comment:
```suggestion
    it('signTransaction', async function () {
```
All of these tests should be renamed accordingly